packages/manager/apps/web/client/app/hosting/cdn/shared/leave-confirm-settings/leave/hosting-shared-leave-settings.html

```diff
@@ -1,91 +1,80 @@
     <div
         class="oui-paragraph"
         data-ng-bind=":: 'hosting_cdn_shared_modal_leave_info' | translate"
     ></div>
 
-    <!--Always online-->
-    <div class="mb-2">
-        <div class="row">
-            <div class="text-right col-md-6">
-                <p class="oui-heading_5 mb-1"><strong>Always online</strong></p>
-            </div>
-            <div class="text-left col-md-6">
-                <p
-                    class="mb-1"
-                    data-ng-bind="::'hosting_cdn_shared_state_' + ($ctrl.model.alwaysOnline.enabled ? 'enable' : 'disabled') | translate"
-                ></p>
-            </div>
+    <dl class="oui-description oui-description_horizontal">
+        <dt><strong>Always online</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.alwaysOnline.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>HTTP/2</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.http.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Devmode</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.devmode.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Brotli</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.brotli.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Cache rule</strong></dt>
+        <dd
+            data-ng-repeat="rule in ::$ctrl.model.rules track by $index"
+            data-ng-bind="::rule.name"
+        ></dd>
+        <div data-ng-if="$ctrl.model.cors">
+            <dt><strong>CORS</strong></dt>
+            <dd>
+                <span
+                    data-ng-if="!$ctrl.model.cors.enabled"
+                    data-translate="hosting_cdn_shared_state_disabled"
+                ></span>
+                <span
+                    data-ng-if="$ctrl.model.cors.enabled"
+                    data-ng-repeat="origin in ::$ctrl.constructor.getCorsOriginsList($ctrl.model.cors.config.origins) track by origin"
+                >
+                    <span data-ng-bind="::origin"></span>
+                    <span data-ng-if="!$last">, </span>
+                </span>
+            </dd>
         </div>
-    </div>
-
-    <!--HTTP/2-->
-    <div class="mb-2">
-        <div class="row">
-            <div class="text-right col-md-6">
-                <p class="oui-heading_5 mb-1"><strong>HTTP/2</strong></p>
-            </div>
-            <div class="text-left col-md-6">
-                <p
-                    class="mb-1"
-                    data-ng-bind="::'hosting_cdn_shared_state_' + ($ctrl.model.http.enabled ? 'enable' : 'disabled') | translate"
-                ></p>
-            </div>
+        <div data-ng-if="$ctrl.model.https_redirect">
+            <dt><strong>HTTPS-Redirect</strong></dt>
```

Comment:
do you think it should come from a constant file? no translation required?

Reply:
Same than for `HSTS`. Also no translation required, as all the options for a CDN will have the same name regardless the user's language